db/views/analytics_cohort_client_changes_v01.sql

```diff
@@ -0,0 +1,2 @@
+SELECT *
```

Comment:
for all views, I think we should enumerate the column names (that's how they end up in the schema sql anyway)

Reply:
Thanks for the reminder!